src/pages/Settings/RecoveryCodeConfirm.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+	<kv-lightbox
+		:visible="visible"
+		title="2-step verification is on"
+		@lightbox-closed="closeLightbox"
+		prevent-close="true"
+	>
+		<p>
+			Copy this recovery code and keep it somewhere safe. You'll
+			need it if you want to log in without your device.
+		</p>
+		<label>
+			<input
```

Comment:
Actually, let's make this a div instead of a label/input, since we just want to display the recovery code that we already have.

Reply:
updated in latest commit